staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go

```diff
@@ -48,6 +47,9 @@ type ListOptions struct {
 	ResourceVersion string
 	// Timeout for the list/watch call.
 	TimeoutSeconds *int64
+
+	Limit    int64
```

Comment:
I guess if we're not returning additional info like total result count there's not much reason to do a limit=0 list